iam/policy-templates/dss-chunked-task-s3_copy_write_metadata-lambda.json

```diff
@@ -0,0 +1,40 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "logs:CreateLogGroup",
+        "logs:CreateLogStream",
+        "logs:DescribeLogStreams",
+        "logs:PutLogEvents"
+      ],
+      "Resource": "arn:aws:logs:*:*:*"
+    },
+    {
+      "Effect": "Allow",
+      "Action": "sns:Publish",
+      "Resource": [
+        "arn:aws:sns:*:$account_id:dss-*-$stage"
+      ]
+    },
+    {
+      "Effect": "Allow",
+      "Action": "s3:*",
```

Comment:
Consider enumerating permissions on a need basis here (to be fair, we may need to do it in other policies too). For example, to limit exposure to catastrophic data loss, we may want to deny delete access to all daemons except a specially managed deleter daemon.

Reply:
done.